modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/util/UpgradeSXPBlueprintContributorStorageSchemaHelper.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.search.experiences.internal.util;
+
+import com.liferay.petra.string.StringPool;
+import com.liferay.search.experiences.model.SXPBlueprint;
+import com.liferay.search.experiences.rest.dto.v1_0.Configuration;
+import com.liferay.search.experiences.rest.dto.v1_0.GeneralConfiguration;
+import com.liferay.search.experiences.rest.dto.v1_0.util.ConfigurationUtil;
+
+import java.util.Objects;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Joshua Cords
+ */
+@Component(
+	enabled = false,
+	service = UpgradeSXPBlueprintContributorStorageSchemaHelper.class
+)
+public class UpgradeSXPBlueprintContributorStorageSchemaHelper {
```

Comment:
Is there a reason to not to have the 'Clause' in the name, like 'UpgradeSXPBlueprintClauseContributorXYZ'?